src/components/common/view/Image.js

```diff
@@ -52,6 +57,10 @@ const Image = ({ style = {}, source, ...props }) => {
     return null
   }
 
+  if (isNativeSvg) {
+    return <Svg uri={uri} width={flattenStyle.width} height={flattenStyle.height} />
+  }
+
   return <NativeImage {...props} source={source} style={imageStyle} />
```

Comment:
again that looks like it will have a completely different look and fool on native than on web
what exactly does this solve? this has always worked for iOS before

Reply:
Native Image has never rendered the svg icons, so we need to use react-native-svg to do it. It doesn't change the design of svg

Reply:
so this is broken for android also?

Reply:
yes, there is no flag on profile screen for android/ios

Reply:
for the svg you dont use imageStyle nor props so anyone using <Image> component will not get expected results.